### DISCUSSION TITLE
Using LAN for RustDesk over two WANs

### DISCUSSION THREAD

**Discussion by Some1Else83129** [rustdesk/rustdesk#5359] [Q&A] [answered]
Good day. I have a router with two ISPs. All of my computers are on the same subnet. Some are directed to ISP 1, some to ISP 2. Different WAN IP addresses.

I'm unsure what version this changed, but if I'm on PC 1 on ISP 1 with LAN IP 192.168.1.20 and PC 2 with 192.168.1.168 - when I connect to the other PC - it goes over the WAN instead of making a direct LAN connection.  
If I change PC 2 to use ISP 1 - it's back to almost instant connections and fast (using LAN I assume).

Is there any settings I can change so PC 1 and 2 talk over LAN instead of WAN when technically on two different ISPs but the same subnet?

Thanks for all you do!

**Comment by SergeyMy** [comment]
connect directly via IP, after enabling this in setting

**Reply by Some1Else83129** [reply]
What is the default port to use? What setting? I liked your idea. I fired up open ports using Resource Monitor on the other PC. TCP 6012 - and some other UDP ports. 
 I attempted "IP:port" in Rustdesk - that did not work - just to make sure "192.168.1.48:6012" - I just want to make sure I'm doing this correct.  That did not connect.

**Comment by SergeyMy** [comment, marked as answer]
there it offers by default :) https://github.com/rustdesk/rustdesk/wiki/FAQ#i-only-use-rustdesk-for-a-few-devices-on-my-local-network-with-no-internet-connectivity-can-i-still-use-rustdesk-with-direct-ip-access
and in the connection string just IP
![изображение](https://github.com/rustdesk/rustdesk/assets/131688106/a06c0599-4e0f-4f74-9816-cbc3cfe2e3e6)

https://github.com/rustdesk/rustdesk/assets/131688106/c1558e70-6bb5-4645-a45d-63aefc1a5a8a

do not forget about the firewall, in normal systems it is enabled :)
and if SELinux is active, oooh, this is something with something

**Reply by Some1Else83129** [reply]
Thank you! I'm all set now.

**Comment by xlionjuan** [comment]
RustDesk's determination need to improve, so it is better to use IP direct connect if it is actually on same LAN.
